high-level-tests/test-algo/tests/tests.sh

```diff
@@ -23,6 +23,7 @@ testExit() {
 }
 
 testExit 'should fetch challenge when competenceId is specified' 'npm start -- --competenceId example_competence_id' 0
+testExit 'should fetch challenge when targetProfileId is specified' 'npm start -- --targetProfileId 1' 0
 testExit 'should return error when no competenceId is specified' "npm start" 1
```

Comment:
```suggestion
testExit 'should return error when no argument is specified' "npm start" 1
```
**Suggestion : ** Je pense que l'on devrait changer la description de ce test vu que l'on peut avoir un targetProfileId sans competenceId

Reply:
Pris en compte